commands/lxc2lxd.go

```diff
@@ -60,3 +62,70 @@ cat << 'EOF' > /var/lib/juju/1.25-upgrade/scripts/lxc-to-lxd
 	}
 	return nil
 }
+
+// StopLXCContainer starts the specified LXC container machine.
+func StartLXCContainer(container, host description.Machine) error {
```

Comment:
I got really confused by this taking `description.Machine`s, not clicking that Stop and Backup took `*state.Machine`s.

Does anything call this? Should restarting the containers be an option? I guess it would make the backups useless if they could be started again before migrating, but maybe for a dry-run.